Jenkinsfile

```diff
@@ -104,7 +114,8 @@ if( buildParams.BUILD_OS != "all" ){
 
 // Using .each{} hits "a CPS-transformed closure is not yet supported (JENKINS-26481)"
 for( label in build_nodes) {
-  stage_unit_test(label)
-  stage_rpmbuild(label)
-  stage_integration(label)
+//stage_unit_test(label)
+//stage_rpmbuild(label)
+//stage_integration(label)
+  stage_clean_rpms(label)
 }
```

Comment:
I mean why this branch has the change to the root ``Jenkinsfile``.  I understand that why ``Jenkinsfile.garbage_collection`` is added and there is new Jenkins job ``garbage_collection`` refers to ``Jenkinsfile.garbage_collection``.